### ISSUE TITLE
Downloading from S3

### ISSUE THREAD

**Issue by mike-robinson** [edvin/fxlauncher#146]
This is probably a misunderstanding on my part, but I am trying to use fxlauncher to download the dependencies from an S3 bucket. The bucket has public read permissions. I have a script that uploads the contents of the app/ directory to the bucket. I can access files in the bucket using a standard url from a browser. However, when I try to have fxlauncher do the check & download, it fails with no message as to why.

As a test, I uploaded the app/ contents to a tomcat server and changed the <app.url> to point to it. Worked just fine.

This is the url I am using for S3 (without the bucket name)
<app.url>http://[mybucket].s3-website-us-west-1.amazonaws.com</app.url>

Can anyone who has successfully done this give me an idea of what might be wrong? Thanks

**Comment by edvin** [comment]
FXLauncher requires nothing special of the http server, so it would be interesting to investigate the actual bucket. Is that possible?

**Comment by mike-robinson** [comment]
I had some thoughts this morning about whether the cache was the problem.
Turns out it was. I think what happened was my experimentation with
different servers and different settings left the cache with some bogus
files and it prevented fxlauncher to connect and/or update with S3.

So I deleted the local cache, rebuild everything and redeployed to S3. Now
it behaves as expected.

I appreciate your offer to help but now it does not seem necessary. Thanks.

Best wishes,
Mike
